app/helpers/application_ui_helper.rb

```diff
@@ -0,0 +1,31 @@
+module ApplicationUiHelper
+  # Create a edit button with icon
+  #
+  # @param path [String] The path of the link.
+  # @param options [Hash] The options to pass to `link_to`
+  # @return [String] The string including embedded HTML which displays
+  #                  the button with a link to path.
+  def edit_button(path, options = {})
+    css_class = options[:class] || ['btn', 'btn-default']
```

Comment:
how about `'btn btn-default'` instead of the array  ? I think we should just use the value user passed in, otherwise this function might become too complicated.